flink-rpc/flink-rpc-akka/src/main/java/org/apache/flink/runtime/rpc/akka/AkkaUtils.java

```diff
@@ -137,7 +137,7 @@ public static Config getThreadPoolExecutorConfig(
         final int maxNumThreads = configuration.getMaxNumThreads();
 
         return new AkkaConfigBuilder()
-                .add("akka {")
+                .add("pekko {")
                 .add("  actor {")
                 .add("    default-dispatcher {")
                 .add("      type = org.apache.flink.runtime.rpc.akka.PriorityThreadsDispatcher")
```

Comment:
That's a bit off-topic (hotfix) but could we add a JavaDoc to this method that mentions `PriorityThreadsDispatcher`? Initially, I got confused by Intellij telling me that `PriorityThreadsDispatcher` is an "unused" class. Adding a descriptive JavaDoc here would make this issue go away.

Reply:
fixed in another way